_docs/drivers.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: page
+title: Drivers
+permalink: /docs/drivers/
+---
+
+As described in the [Automation Pipeline](../docs/overview) document, a driver
+must be present on the same machine as the browser you are automating.
+
+If you are running locally, you can use the [webdrivers gem](https://github.com/titusfortner/webdrivers)
+to automatically ensures that the latest driver is downloaded, and placed in a location
+where Selenium can access it.
+
+Otherwise, you can download the drivers for the browser of your choice:
+* Google Chrome: [chromedriver](https://sites.google.com/a/chromium.org/chromedriver/downloads)
+* Mozilla Firefox: [geckodriver](https://github.com/mozilla/geckodriver/releases)
+* Microsoft Edge: [Microsoft WebDriver](https://github.com/mozilla/geckodriver/releases)
+* Microsoft Internet Explorer: [IEDriver](http://docs.seleniumhq.org/download)
+* Apple Safari: safaridriver (no download needed, it's pre-installed if you have Safari 10)
+
+For all browsers except for Safari, you then need to move the driver onto your PATH. 
```

Comment:
This might be mixing the pot a bit, but I've been using https://github.com/vvo/selenium-standalone to stand up selenium and the drivers locally. I know it's JS, but it's really just a CLI tool.